apps/nextra/pages/en/build/cli/formatting-move-contracts.mdx

```diff
@@ -0,0 +1,75 @@
+
+# Formatting Move Contracts
+
+`movefmt` is a formatter tool that makes Move code much easier to write, read, and maintain — greatly improving the development experience on Aptos.
+
+## Installation
+
+`movefmt` is integrated into the Aptos CLI. To begin using it, first install it using the CLI update command.
+
+```bash filename="Terminal"
+# Install movefmt for first time usage
+aptos update movefmt
+```
+
+When a newer version of `movefmt` is available, use the following command to install it:
+
+```bash filename="Terminal"
+# Install movefmt with the target <VERSION>
+aptos update movefmt --target-version <VERSION>
```

Comment:
Unfortunately, the current updater does not support updating to the latest version with out input. Instead, it will install the version hard coded

Reply:
Will we be updating the hard coded version whenever a new version is available?

If not, when will the hardcoded version be updated?

Reply:
We should do it once the formatter is more stable but since it is still beta and the update will be frequent, it is harder to catch up with it. I think we can do it before each CLI release.

Reply:
Awesome, makes sense.